src/compiler/transformers/map-imports-to-path-aliases.ts

```diff
@@ -0,0 +1,58 @@
+import { dirname, relative } from 'path';
+import ts from 'typescript';
+import type * as d from '../../declarations';
+
+export const mapImportsToPathAliases = ({ tsCompilerOptions }: d.Config): ts.TransformerFactory<ts.SourceFile> => {
+  return (transformCtx) => {
+    let dirPath: string;
+    let sourceFile: string;
+
+    const compilerHost = ts.createCompilerHost(tsCompilerOptions);
+
+    const visit = (node: ts.Node): ts.VisitResult<ts.Node> => {
+      if (ts.isImportDeclaration(node) && ts.isStringLiteral(node.moduleSpecifier)) {
+        let importPath = node.moduleSpecifier.text;
+
+        if (!importPath.startsWith('.')) {
+          /**
+           * When running unit tests, the `resolvedModule` property on the returned object is always undefined.
+           * In an actual build, the modules are resolved as expected.
+           *
+           * Not sure what is causing this to fail in tests. My guess is that the `transpileModule` helper method
+           * is somehow not giving context to paths for the options supplied on a `config.tsCompilerOptions`, but I am confused
+           * as to how this differs between tests and a build.
+           */
+          const module = ts.resolveModuleName(importPath, sourceFile, tsCompilerOptions, compilerHost);
```

Comment:
This is what is causing tests to fail for this transformer. In the testing context, TS is not able to resolve the import modules, so import paths are always left untouched.